components/server/src/data/datamodel.json

```diff
@@ -4726,6 +4729,110 @@
             },
             "entities": {}
         },
+        "generic": {
+            "name": "JSON file with Security Warnings",
+            "description": "Generic Security Warning vulnerability report in JSON format",
+            "url": "https://github.com/ICTU/quality-time/blob/master/docs/GENERIC_REPORTS.MD",
+            "parameters": {
+                "url": {
+                    "name": "URL to a generic vulnerability report in JSON format",
+                    "short_name": "URL",
+                    "type": "url",
+                    "mandatory": true,
+                    "default_value": "",
+                    "validate_on": [
+                        "username",
+                        "password",
+                        "private_token"
+                    ],
+                    "metrics": [
+                        "security_warnings"
+                    ]
+                },
+                "landing_url": {
+                    "name": "URL to a Generic vulnerability report in a human readable format",
+                    "short_name": "URL",
+                    "help": "If provided, users clicking the source URL will visit this URL instead of the Generic vulnerability report in JSON format.",
```

Comment:
```suggestion
                    "help": "If provided, users clicking the source URL will visit this URL instead of the generic vulnerability report in JSON format.",
```